lib/directives/mobx-autorun-sync.directive.ts

```diff
@@ -1,14 +1,18 @@
 import { Directive, ViewContainerRef, TemplateRef } from '@angular/core';
 import { autorun } from 'mobx';
-import {MobxAutorunDirective} from './mobx-autorun.directive';
+import { MobxAutorunDirective } from './mobx-autorun.directive';
 
 @Directive({ selector: '[mobxAutorunSync]' })
 export class MobxAutorunSyncDirective extends MobxAutorunDirective {
   constructor(
     protected templateRef: TemplateRef<any>,
-    protected viewContainer: ViewContainerRef) {super(templateRef, viewContainer); }
+    protected viewContainer: ViewContainerRef
+  ) {
+    super(templateRef, viewContainer);
+  }
 
   autoDetect(view) {
+    // prettier-ignore
     console.warn('mobxAutorunSync is deprecated, please use mobxAutorun instead - it\'s doing exactly the same thing');
```

Comment:
Prettier and ESlint are conflicting are. We can ignore it because the MobxAutorunSyncDirective will be eventually removed: https://github.com/mobxjs/mobx-angular/issues/102